circuits/cpp/barretenberg/cpp/src/barretenberg/dsl/acir_format/serde/acir.hpp

```diff
@@ -4494,6 +4458,7 @@ void serde::Serializable<Circuit::Circuit>::serialize(const Circuit::Circuit& ob
     serializer.increase_container_depth();
     serde::Serializable<decltype(obj.current_witness_index)>::serialize(obj.current_witness_index, serializer);
     serde::Serializable<decltype(obj.opcodes)>::serialize(obj.opcodes, serializer);
+    serde::Serializable<decltype(obj.private_parameters)>::serialize(obj.private_parameters, serializer);
```

Comment:
I'm surprised the decltype pattern is needed here vs deriving the type from the first arg